### DISCUSSION TITLE
Fedora/RHEL support

### DISCUSSION THREAD

**Discussion by calculatetech** [himmelblau-idm/himmelblau#274] [Ideas]
I'm voicing my interest in support for Fedora.  Sadly, I do not know how to make that contribution to the project.

**Comment by dmulder** [comment]
I've added the ability to build EL8/9 (Rocky) in 0.7.x, there just aren't any published versions yet. I haven't done Fedora yet, but that will be trivial.

**Comment by dmulder** [comment]
I'm adding Fedora to the main branch now: https://github.com/himmelblau-idm/himmelblau/pull/285
Rocky 8/9 are now published via https://github.com/himmelblau-idm/himmelblau/releases/tag/0.7.2
